lib/chef_zero/server.rb

```diff
@@ -291,9 +293,17 @@ def start_background(wait = 5)
       ENV['HTTPS'] = 'on' if options[:ssl]
       @server.mount('/', Rack::Handler::WEBrick, app)
 
+      # If options[:port] can be an Enumerator, an Array, or an Integer,
+      # we need something that can respond to .each (Enum and Array can already).
+      enumerable = if options[:port].respond_to?(:each)
```

Comment:
If you just say `Array(options[:port], that will do what you need without an `if` statement.